package.json

```diff
@@ -127,7 +129,7 @@
     }
   },
   "engines": {
-    "node": ">= 10.*"
+    "node": ">= 14.*"
```

Comment:
When `ember-intl@v6.0.0` is released, we may want to denote that Node 10 and 12 supports are dropped?